weave

```diff
@@ -999,17 +999,17 @@ wait_for_status() {
 
 # Call $1 for all containers, passing container ID, all MACs and all IPs
 with_container_addresses() {
-    COMMAND=$1
+    COMMAND_WCA=$1
```

Comment:
The point of this commit is to avoid having the same variable name in three places, where it can overwrite if called re-entrantly.  So changing two of them to `CMD` leaves us with the same danger.

The name is perhaps ugly but was deliberately chosen to avoid clashes.